packages/spruce-types/src/gql/types/SprucebotAvatar.gql

```diff
@@ -0,0 +1,32 @@
+"Sprucebot's current state of mind"
+enum SprucebotAvatarStateOfMind {
+	"For when nothing in particular is happening"
+	chilling
+	"When submitting a form or running some other operation"
+	thinking
+	"Asking a question or leaning new things"
+	inquisiting
+	"Something has finished and we're so excited!"
+	jamming
+}
+
+enum SprucebotAvatarSize {
+	"Inline with text"
+	small
+	"A subheading"
+	medium
+	"A heading"
+	large
+}
+
+"Your friendly neighborhood Sprucebot Avatar!"
+type SprucebotAvatar {
+	"ID for view caching"
```

Comment:
Eh, don't think we need this. IDs are used for other reasons and people will generally understand what an ID is anyway.

Reply:
Alternatively it can just be generic "A unique identifier for this record"

Reply:
@gcpantazis - I had been making ID's required for all components so we never run the risk of showing a "you must set a key on a component when looping" or whatever that error is.

Reply:
Sorry, I meant keep `id` but drop the specific comment "ID for view caching" since IDs are more generic than that.